.claude/skills/mcp-server.md

```
@@ -0,0 +1,105 @@
+
+# Start
+bash -lc "cd \$(git rev-parse --show-toplevel) && ./scripts/run_server_with_token.sh >/tmp/mcp_agent_mail_server.log 2>&1 & echo \$!"
+
+# Wait and verify
+sleep 2
+curl -s http://127.0.0.1:8765/health | jq '.'
+```
+
+## Server Details
+
+- **Endpoint:** http://127.0.0.1:8765/mcp/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)